scapy/layers/bluetooth.py

```diff
@@ -91,35 +104,11 @@ def mysummary(self):
 
 class HCI_ACL_Hdr(Packet):
     name = "HCI ACL header"
-    fields_desc = [BitField("handle", 0, 12),    # TODO: Create and use LEBitField  # noqa: E501
-                   BitField("PB", 0, 2),      # They are received as a **combined** LE Short  # noqa: E501
-                   BitField("BC", 0, 2),      # Handle is 12 bits, each flag is 2 bits.  # noqa: E501
+    fields_desc = [BitField("BC", 0, -2),
+                   BitField("PB", 0, -2),
+                   BitField("handle", 0, -12),
```

Comment:
This will surely fail at first, I think we could get rid of the pre_post metric thanks to this